src/java/org/apache/cassandra/db/marshal/TypeParser.java

```diff
@@ -131,6 +138,56 @@ public AbstractType<?> parse() throws SyntaxException, ConfigurationException
             return getAbstractType(name);
     }
 
+    /**
+     * Parse PartitionOrdering from old version of PartitionOrdering' string format 
+     * */
```

Comment:
Same as in the review for 3.0 and my previous round for trunk: this should be `*/`:
```suggestion
     */
```